IBF-Typhoon-model/mainpipeline.py

```diff
@@ -34,120 +34,91 @@
 #path='C:/Users/ATeklesadik/OneDrive - Rode Kruis/Documents/Typhoon-Impact-based-forecasting-model/IBF-Typhoon-model/'
 #path='/home/fbf/'
 path = './'
-
```

Comment:
Is this file used anymore? It's awfully similar to pipeline.py. If it's not used then it should be removed